src/main/java/cn/nukkit/item/ItemNameTag.java

```diff
@@ -0,0 +1,32 @@
+package cn.nukkit.item;
+
+import cn.nukkit.entity.Entity;
+import cn.nukkit.entity.EntityHuman;
+import cn.nukkit.entity.mob.EntityEnderDragon;
+
+public class ItemNameTag extends Item {
+
+    public ItemNameTag() {
+        this(0, 1);
+    }
+
+    public ItemNameTag(Integer meta) {
+        this(meta, 1);
+    }
+
+    public ItemNameTag(Integer meta, int count) {
+        super(NAME_TAG, meta, count, "Name Tag");
+    }
+
+    @Override
+    public boolean useOn(Entity entity) {
```

Comment:
Name tag behavior is already implemented (see `EntityMob` and `EntityAnimal`). If you want to move it here make sure it's not there in the both places.